.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 dist
-.idea
\ No newline at end of file
+.idea
```

Comment:
Also you can add this to your global gitignore IF you don't want IDE specific objects in the repo gitignore

Reply:
Thanks. It was already in the file. I just added the screenshots/ directory.

Reply:
@krobi64 we're commiting IDE configuration git-ignore to prevent anyone from accidentally commiting it